ffi/diplomat/c/examples/fixeddecimal_tiny/icu_capi_staticlib_tiny/src/lib.rs

```diff
@@ -9,18 +9,12 @@ extern crate icu_capi;
 
 extern crate dlmalloc;
 
-use core::alloc::Layout;
 use core::panic::PanicInfo;
 use dlmalloc::GlobalDlmalloc;
 
 #[global_allocator]
 static ALLOCATOR: GlobalDlmalloc = GlobalDlmalloc;
 
-#[alloc_error_handler]
```

Comment:
Question: what was this replaced with?

Reply:
It defaults to a panic pre-yesterday's nightly, and after yesterday's nightly it is _always_ a panic, with the important stuff in the panic metadata